### ISSUE TITLE
Cannot create a default list of maps for a variable

### ISSUE THREAD

**Issue by oillio** [hashicorp/terraform#7032]
Is there a new syntax to create a default list of maps for the new list variables?

I can pass in a list of maps using this syntax:

```hcl
module "envTesting" {
  source = "../modules"
  foo {
    value     = "aVal"
  }
  foo {
    value     = "anotherVal"
  }
}
```

However, I cannot find a way to define a default list of maps for a variable.

### Terraform Version
Terraform v0.7.0-rc1 (301da85f30239e87b30db254a25706a6d41c2522)

**Comment by jen20** [comment]
Hi @oillio! Thanks for opening an issue and testing out the release candidates! This isn't currently supported in Terraform 0.7, but will be at some point in the future. The infrastructure is there to support it in interpolations, but state needs more work in order to support this correctly.

**Comment by oillio** [comment]
Fixed in #7582
